modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/validation/ParameterModelValidator.java

```diff
@@ -143,6 +181,23 @@ private void validateParameter(ParameterModel parameterModel, MetadataTypeVisito
     }
   }
 
+  private void validateParameterFieldsHaveGetters(ComponentModel model, Class<?> parameterType) {
+    String modelTypeName = getComponentModelTypeName(model);
+    String modelName = model.getName();
+    Collection<Field> parameters = getAnnotatedFields(parameterType, Parameter.class);
```

Comment:
Do not introspect again. Use the information already present in the MetadataType.